docker/dev/docker-compose.yml

```diff
@@ -20,7 +20,7 @@ services:
       APP_NAME: Engelsystem DEV
     env_file: deployment.env
     ports:
-      - "5000:80"
+      - "5001:80"
```

Comment:
To be consistent that should be changed in `docker/docker-compose.yml` too (and added to the dev documentation as it is missing there ^^).

Reply:
done